src/Makefile.am

```diff
@@ -1,6 +1,7 @@
 USB_MODED_CFLAGS += -D_GNU_SOURCE \
 	$(DBUS_CFLAGS)
 USB_MODED_LIBS   += -lpthread
+USB_MODED_LIBS	 += -lsystemd
```

Comment:
Doesn't `--enable-systemd` (like seen on https://gitlab.com/ubports/development/core/packaging/usb-moded/-/blob/ubports/latest/debian/rules#L18) during configure practically do the same without any changes here?

Reply:
Oh I missed that